electron_app/src/electron-main.js

```diff
@@ -232,6 +232,10 @@ ipcMain.on('ipcCall', async function(ev, payload) {
             }
             break;
         case 'origin_migrate':
+            // NB. The code to call this has now been removed so this
+            // should never be called. This code itself has been removed
+            // with the migration to the riot-desktop repo, with the
+            // expectation that this will become the new home for this code.
```

Comment:
Hmm, even after reading this comment, I am still puzzled as to why we're leaving this `origin_migrate` message handler in place now that nothing calls it... Will we remove it at some future time?

Reply:
Sorry, have tried to explain better! We could also just remove the code in both places, it's not actually very invasive.

Reply:
Hmm, I guess it just feels strange to remove it half-way like this... What if we end up abandoning the separate repo experiment for some reason? Then we'll be left with this half-way removed thing that we're likely to forget about.

I'd suggest removing the message handler and the module over here as well.

Reply:
Fair enough - removed here too